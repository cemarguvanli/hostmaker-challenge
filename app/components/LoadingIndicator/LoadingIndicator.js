import React from 'react';
import styles from './styles.scss';

const LoadingIndicator = () => (
  <div className={styles.loadingIndicator}>
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
);

export default LoadingIndicator;
