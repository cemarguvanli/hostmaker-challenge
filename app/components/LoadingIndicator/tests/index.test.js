import React from 'react';
import { render } from 'enzyme';
import renderer from 'react-test-renderer';

import LoadingIndicator from '../index';

describe('<LoadingIndicator />', () => {
  it('should render 8 divs', () => {
    const renderedComponent = render(<LoadingIndicator />);
    const divs = renderedComponent.find('div');
    expect(divs.length).toEqual(8);
  });

  it('renders correctly', () => {
    const tree = renderer
      .create(<LoadingIndicator />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
