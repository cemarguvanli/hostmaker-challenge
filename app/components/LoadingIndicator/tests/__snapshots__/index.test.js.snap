// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingIndicator /> renders correctly 1`] = `
<div>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
